b2b-sample/apps/business-admin-app/components/sections/home.tsx

```diff
@@ -16,36 +16,34 @@
  * under the License.
  */
 
+import { LogoComponent } from "@b2bsample/business-admin-app/ui/ui-components";
+import { signout } from "@b2bsample/business-admin-app/util/util-authorization-config-util";
+import { FooterComponent, HomeComponent, SignOutComponent } from "@b2bsample/shared/ui/ui-components";
 import React, { useState } from "react";
 import "rsuite/dist/rsuite.min.css";
-import SideNavSection from "./otherComponents/sideNavSection";
-import DashboardSectionComponent from "./sections/dashboardSection/dashboardSectionComponent";
 import IdpSectionComponent from "./sections/settingsSection/idpSection/idpSectionComponent";
 import ManageUserSectionComponent from "./sections/settingsSection/manageUserSection/manageUserSectionComponent";
 import RoleManagementSectionComponent from
     "./sections/settingsSection/roleManagementSection/roleManagementSectionComponent";
+import sideNavData from "../../../../libs/business-admin-app/ui/ui-assets/src/lib/data/sideNav.json";
 import Custom500 from "../../pages/500";
-import styles from "../../styles/Settings.module.css";
-import SignOutModal from "../common/signOutModal";
+
```

Comment:
unwanted new line. Please check other places also.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/3e0450172639aa334cdc6716a7e8640db58efa19